docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -183,408 +118,126 @@ your Identity Schema:
+field. The only thing the user must do to verify their address is to submit the form.
 
-The new Verification Flow includes an error message such as the following:
+```mdx-code-block
+  </TabItem>
+  <TabItem value="link" label="Magic Links">
+```
 
-<CodeTabs items={getFlowMethodLinkInvalidChallenge} />
+When using the "magic link" method, Ory Identities sends the user an email with a verification link. If the user has access to the
+email associated with the account and clicks the link, Ory Identities marks that address as verified.
 
-Please keep in mind that this part of the flow always involves a Browser!
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
-## Successful verification
+If the user completes the verification after the link or code expired, they must provide the email address associated with their
+account to get a new verification email.
 
-If the verification challenge is completed (for example the sent verification link was clicked), the end-user's Browser is
-redirected to the Verification UI with a Verification Flow that has now the `state` of `passed_challenge`:
+### Comparison
 
-<CodeTabs items={getFlowMethodLinkChallengeDone} />
+```mdx-code-block
+import CodeLinkComparison from "./_common/code-link-comparison.mdx"
 
-You may also [configure a redirect URL](../../concepts/browser-redirect-flow-completion.mdx) instead which would send the end-user
-to that configured URL.
+<CodeLinkComparison />
+```
 
-## Code examples for Node.js, React.js, Go, ...
+## Code examples
 
-The Verification User Interface is a route (page / site) in your application (server, native app, single page app) that should
-render a verification form.
+The user interface for account verification is a page in your solution that renders the actual form elements for the user.
 
-In stark contrast to other Identity Systems, Ory Identities doesn't render this HTML. Instead, you need to implement the HTML code
-in your application (for example Node.js + Express.js, Java, PHP, React.js, ...), which gives you extreme flexibility and
-customizability in your user interface flows and designs.
+In contrast to other identity systems, Ory Identities (Ory Kratos) doesn't render this HTML directly. Instead, you need to
+implement the HTML code in your solution, which gives you complete flexibility and customizability in your user interface flows
+and designs. This part of your application then directly interfaces with Ory Identities through the API.
 
-You will use the Verification Flow JSON response to render the verification form UI, which could looks as follows depending on
-your programming language and web framework:
+The API responds with a JSON document describing the form elements to render and actions the form should take upon submission,
+cancellation, etc. The following shows examples for a few different languages and frameworks.
 
 <RenderFlow flow="verification" />
```

Comment:
This is weird - that "Code examples" thing is now text only. The `<RenderFlow flow="verification" />` doesn't seem to do anything. Can you see what's going on here?

Reply:
Whoops, removed the import a bit prematurely, but the tooling didn't complain so I didn't notice. Good catch.